src/App.js

```diff
@@ -17,6 +17,7 @@ import Join from 'components/Join';
 import LoginPage from 'components/LoginPage';
 import FileEntity from './components/EntityPage/File';
 import ParticipantEntity from './components/EntityPage/Participant';
+import CohortBuilder from './components/CohortBuilder';
```

Comment:
🔥 :)

Ciaran is looking at hooking this up with lazy loading actually :) just wanna remove potential conflicts